MapboxNavigation/FeedbackItem.swift

```diff
@@ -16,71 +16,71 @@ public extension FeedbackType {
         case .illegalRoute(.routedDownAOneWay):
-            return "Routed down a one way"
+            return NSLocalizedString("ROUTE_QUALITY_ILLEGAL_ROUTE_ONE_WAY_FEEDBACK", bundle: .mapboxNavigation, value: "Routed down a one way", comment: "Specific route feedback that route travelled wrong way on a one-way road.")
         case .illegalRoute(.turnWasNotAllowed):
-            return "Turn wasn't allowed"
+            return NSLocalizedString("ROUTE_QUALITY_ILLEGAL_ROUTE_TURN_NOT_ALLOWED_FEEDBACK", bundle: .mapboxNavigation, value: "Turn wasn't allowed", comment: "Specific route feedback that route suggested an illegal turn.")
         case .illegalRoute(.carsNotAllowedOnStreet):
-            return "Cars not allowed on street"
+            return NSLocalizedString("ROUTE_QUALITY_ILLEGAL_ROUTE_CARS_NOT_ALLOWED_FEEDBACK", bundle: .mapboxNavigation, value: "Cars not allowed on street", comment: "Specific route feedback that route suggested an illegal roadway.")
         case .illegalRoute(.turnAtIntersectionUnprotected):
-            return "Turn at intersection unprotected"
+            return NSLocalizedString("ROUTE_QUALITY_ILLEGAL_ROUTE_UNPROTECTED_TURN_FEEDBACK", bundle: .mapboxNavigation, value: "Turn at intersection unprotected", comment: "Specific route feedback that route suggested an illegal unprotected turn.")
         case .illegalRoute(.other):
-            return "Other"
+            return NSLocalizedString("ILLEGAL_ROUTE_OTHER_FEEDBACK", bundle: .mapboxNavigation, value: "Other", comment: "Specific route feedback that a illegal route problem was encountered but not listed as a choice.")
         case .roadClosure(.none):
-            return "Road closure"
+            return NSLocalizedString("ROUTE_QUALITY_ROAD_CLOSURE_FEEDBACK", bundle: .mapboxNavigation, value: "Road closure", comment: "General route feedback that route contained closed road.")
         case .roadClosure(.streetPermanentlyBlockedOff):
-            return "Street permanently blocked off"
+            return NSLocalizedString("ROUTE_QUALITY_ROAD_CLOSURE_PERMANENT_FEEDBACK", bundle: .mapboxNavigation, value: "Street permanently blocked off", comment: "Specifc route feedback that route contained road that is permanently closed.")
         case .roadClosure(.roadMissingFromMap):
-            return "Road missing from map"
+            return NSLocalizedString("ROUTE_QUALITY_ROAD_MISSING_FROM_MAP_FEEDBACK", bundle: .mapboxNavigation, value: "Road missing from map", comment: "Specifc route feedback that route contained road not shown on map.")
         case .roadClosure(.other):
-            return "Other"
+            return NSLocalizedString("ROAD_CLOSURE_OTHER_FEEDBACK", bundle: .mapboxNavigation, value: "Other", comment: "Specific route feedback that a road closure type was encountered but not listed as a choice.")
```

Comment:
This was probably a pain to make localizable. As tail work, to reduce verbosity, we should move these strings into a localizable plist. That would also bring us a little closer to making the feedback categories more configurable.